data/rule_data.yml

```diff
@@ -110,3 +110,48 @@ rule_data:
   # https://enterprisecontract.dev/docs/ec-policies/release_policy.html#test_package
   informative_tests:
   - sast-snyk-check
+
+  disallowed_packages:
+  # Disallow hashicorp packages with restrictive licenses.
+  - purl: pkg:golang/github.com/hashicorp/terraform
+    format: semverv
+    # https://github.com/hashicorp/terraform/blob/91488b334e230398765438dce7e90b7db9af0dac/LICENSE#L7
+    min: v1.6.0
```

Comment:
Sorry for the dumb question, but `min: v1.6.0` means everything before `v1.6.0` is allowed?

Reply:
That's right! This specifies the list of disallowed packages. Anything that matches it is disallowed, anything that doesn't is allowed. (Ranges are inclusive.)